src/main/java/racingcar/InputView.java

```diff
@@ -0,0 +1,23 @@
+package racingcar;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    public InputView() {
+    }
+
+    public int createCarNum() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("자동차 대수는 몇 대 인가요?");
+        int carNum = scanner.nextInt();
+        return carNum;
```

Comment:
```suggestion
        return scanner.nextInt();
```
이런 경우는 바로 반환해주는 건 어떨까요~?